WireDomain/Sources/WireDomain/Repositories/Connections/ConnectionsRepository.swift

```diff
@@ -70,7 +70,7 @@ public struct ConnectionsRepository: ConnectionsRepositoryProtocol {
             await withThrowingTaskGroup(of: Void.self) { taskGroup in
                 for connection in connections {
                     taskGroup.addTask {
-                        try await connectionsLocalStore.storeConnection(connection)
+                        try await connectionsLocalStore.storeConnection(connection.toDomainModel())
```

Comment:
prepare data for the store by mapping it to a domain model (so the local store doesn't know about the API layer)